pytorch_lightning/utilities/seed.py

```diff
@@ -41,15 +41,17 @@ def seed_everything(seed: Optional[int] = None) -> int:
 
     try:
         if seed is None:
-            seed = os.environ.get("PL_GLOBAL_SEED", _select_seed_randomly(min_seed_value, max_seed_value))
+            if "PL_GLOBAL_SEED" in os.environ:
+                seed = os.environ["PL_GLOBAL_SEED"]
+            else:
+                seed = _select_seed_randomly(min_seed_value, max_seed_value)
         seed = int(seed)
     except (TypeError, ValueError):
         seed = _select_seed_randomly(min_seed_value, max_seed_value)
 
-    if (seed > max_seed_value) or (seed < min_seed_value):
+    if not (seed < min_seed_value <= seed <= max_seed_value):
```

Comment:
This is wrong, isnt it?

```suggestion
    if not (min_seed_value <= seed <= max_seed_value):
```